app/package.json

```diff
@@ -13,33 +13,34 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/common": "2.0.0-rc.1",
-    "@angular/compiler": "2.0.0-rc.1",
-    "@angular/core": "2.0.0-rc.1",
-    "@angular/http": "2.0.0-rc.1",
-    "@angular/platform-browser": "2.0.0-rc.1",
-    "@angular/platform-browser-dynamic": "2.0.0-rc.1",
-    "@angular/router": "2.0.0-rc.1",
-    "@angular/router-deprecated": "2.0.0-rc.1",
-    "@angular2-material/button": "2.0.0-alpha.4",
-    "@angular2-material/card": "2.0.0-alpha.4",
-    "@angular2-material/checkbox": "2.0.0-alpha.4",
-    "@angular2-material/core": "2.0.0-alpha.4",
-    "@angular2-material/grid-list": "2.0.0-alpha.4-3",
-    "@angular2-material/icon": "2.0.0-alpha.4",
-    "@angular2-material/input": "2.0.0-alpha.4",
-    "@angular2-material/list": "2.0.0-alpha.4",
-    "@angular2-material/progress-bar": "2.0.0-alpha.4",
-    "@angular2-material/progress-circle": "2.0.0-alpha.4",
-    "@angular2-material/radio": "2.0.0-alpha.4",
-    "@angular2-material/sidenav": "2.0.0-alpha.4",
-    "@angular2-material/toolbar": "2.0.0-alpha.4",
-    "es6-shim": "^0.35.0",
-    "moment": "^2.13.0",
-    "reflect-metadata": "0.1.3",
+    "@angular/common": "2.0.0-rc.4",
+    "@angular/compiler": "2.0.0-rc.4",
+    "@angular/core": "2.0.0-rc.4",
+    "@angular/http": "2.0.0-rc.4",
+    "@angular/platform-browser": "2.0.0-rc.4",
+    "@angular/platform-browser-dynamic": "2.0.0-rc.4",
+    "@angular/router": "3.0.0-beta.1",
```

Comment:
I believe the latest router version is actually [rc.1](https://github.com/angular/angular/blob/master/modules/@angular/router/CHANGELOG.md#300-rc1-2016-08-09).

**_Note:_** Angular RC5 was released yesterday and it introduces new global concept `NgModules`. it's a _breaking change_ and I think [rc.1](https://github.com/angular/angular/blob/master/modules/@angular/router/CHANGELOG.md#300-rc1-2016-08-09) is actually expecting you to run Angular RC5. You may either decide to upgrade both [Angular](https://angular.io/docs/ts/latest/cookbook/rc4-to-rc5.html) and router or use router [beta.2](https://github.com/angular/angular/blob/master/modules/@angular/router/CHANGELOG.md#300-beta2-2016-06-30)